packages/backend-core/src/cache/writethrough.ts

```diff
@@ -47,7 +47,7 @@ async function put(
         type: LockType.TRY_ONCE,
         name: LockName.PERSIST_WRITETHROUGH,
         resource: key,
-        ttl: 1000,
+        ttl: 15000,
```

Comment:
This TTL is only in place to ensure that the lock does not get stuck forever. It can virtually be anything, but 15s should be more than enough for the current operations